tests/llm/test_vertex_ai.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from unittest.mock import MagicMock, patch
```

Comment:
It's fine to use patch from unittest package for now but lets change everything to use `pytest-mock` going forward.

Reply:
Got it!